src/docs/development/ui/layout/index.md

```diff
@@ -11,28 +11,31 @@ description: Learn how Flutter's layout mechanism works and how to build a layou
 <style>dl, dd { margin-bottom: 0; }</style>
 
 {{site.alert.secondary}}
-  <h4 class="no_toc">What you’ll learn</h4>
+  <h4 class="no_toc">Flutter Essentials</h4>
 
   * How Flutter's layout mechanism works.
-  * How to lay out widgets vertically and horizontally.
-  * How to build a Flutter layout.
+  * How to lay out widgets vertically or horizontally.
+  * How to build a Flutter layout of widgets
+  * Always rember that even a padding is a widget (of widgets)
+  
 {{site.alert.end}}
 
-This is a guide to building layouts in Flutter.
-You'll build the layout for the following app:
```

Comment:
@pedromorgan, this "you" isn't a problem, is it? I agree, 100%, that "your widget" or "our container" is heinous and not good for translation, but this is telling the user what he or she will be doing in this tutorial.

Reply:
Also, an unadorned "this" is not recommended. See the "this, that" entry in the wordlist:  https://developers.google.com/style/word-list#letter-T